packages/flutter/test/material/app_test.dart

```diff
@@ -6,7 +6,7 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:flutter/material.dart';
```

Comment:
Unfortunately dartfmt doesn't implement the more aggressive style guide we use in the flutter repo.
You can see the style guide here: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
We have a stricter style because readability is paramount for the flutter repository since lots of people are going to read this code. I'm hoping to have a formatter for this style at some point but haven't yet gotten around to finishing that... See https://github.com/flutter/flutter/issues/2025